packages/functional-tests/tests/misc/cookiesDisabled.spec.ts

```diff
@@ -18,53 +18,20 @@ test.describe('cookies disabled', () => {
     //Goto cookies disabled url
     await page.goto(`${target.contentServerUrl}?disable_local_storage=1`);
 
-    //Adding the timeout as the page closes before loading
-    await page.waitForTimeout(2000);
-
     //Verify the Cookies disabled header
     await page.waitForURL(/\/cookies_disabled/);
     await expect(await cookiesDisabled.cookiesDisabledHeader()).toBeVisible();
 
     //Click retry
     await cookiesDisabled.clickRetry();
+    // this is needed in this test but manual clicking once works just fine in
+    // a manual test so
+    await cookiesDisabled.clickRetry();
 
     //Verify the error
     await expect(await cookiesDisabled.isCookiesDiabledError()).toBeVisible();
   });
 
-  test('synthesize enabling cookies by visiting the enter email page, then cookies_disabled, then clicking "try again', async ({
-    target,
-    page,
-    pages: { cookiesDisabled, signin },
-  }) => {
```

Comment:
In addition to being a pain since we fake the pref with a query param, this test (and I think the "try again" button itself really) doesn't even make sense.  When I update my Firefox preferences to enable/disable cookies, the browser forces me to reload all the tabs anyway.

Reply:
I got the impression that this test wasn't working at all since it didn't catch that this has been broken for a long time. Kinda wonder if it's even worth trying to fix?

Reply:
@nshirley for ref, some testing issues with cookies disabled state. I'll ping you on Slack to discuss what the Jira task should be here.

Reply:
Filed a ticket: https://mozilla-hub.atlassian.net/browse/FXA-11486